app/components/common/CorneredContainer.js

```diff
@@ -60,18 +60,19 @@ type Props = {
   header: string,
   headerIcon?: Object,
   subHeader?: string,
+  hasError?: boolean,
   useEmptyWrap: boolean
 };
 
 class CorneredContainer extends PureComponent<Props> {
   render() {
-    const { children, width, height, header, headerIcon, subHeader, useEmptyWrap } = this.props;
+    const { children, width, height, header, headerIcon, subHeader, hasError, useEmptyWrap } = this.props;
```

Comment:
change 'hasError' to 'isErrorModal', more meaningful name

Reply:
fix